### PR TITLE
Small refactors

### PR DESCRIPTION
**Pull request by nozwock**
## Feature
- Add app name to defines
- App name gets version from environment

## Refactor
- Rename `constants.rs` to `defines.rs`

## Misc
- Clean up comments
- Clean up imports

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -4,26 +4,26 @@
...
+use defines::*;
 use eframe::egui;
 
-// When compiling natively:
+// When compiling natively
 #[cfg(not(target_arch = "wasm32"))]
 fn main() {
     let native_options = eframe::NativeOptions {
         always_on_top: true,
-        decorated: true,
```
Comment (MrTanoshii) [inline src/main.rs:18]: Let's keep this for now

**Review by MrTanoshii — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 1 above)
Comment (nozwock) [reply src/main.rs]: Done

**Comment by MrTanoshii** [comment]
Contributes to #3

**Comment by nozwock** [comment]
Thanks for the better description

**Review by MrTanoshii — APPROVED** [review]

**Pull request merged**